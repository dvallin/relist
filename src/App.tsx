import * as React from 'react'
import { hot } from 'react-hot-loader/root'

import Counter from './Counter'

const App = () => (
    <div>
        <h1>Hello, World!</h1>
        <Counter />
    </div>
)

export default hot(App)
