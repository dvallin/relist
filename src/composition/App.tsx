import * as React from 'react'
import { hot } from 'react-hot-loader/root'

import ListSelector from '../lists/ListSelector'
import SelectedList from '../lists/SelectedList'

export const App = () => (
    <div>
        <ListSelector />
        <SelectedList />
    </div>
)

export default hot(App)
