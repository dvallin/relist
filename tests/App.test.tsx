import * as ReactDOM from 'react-dom'
import * as TestUtils from 'react-dom/test-utils'
import App from '../src/App'

describe(App.name, () => {
    it('renders title', () => {
        // when
        const element = TestUtils.renderIntoDocument(App()) as Element

        const node = ReactDOM.findDOMNode(element)!

        expect(node).toMatchSnapshot()
    })
})
