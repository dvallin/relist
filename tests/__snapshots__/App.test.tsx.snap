// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders title 1`] = `
<div>
  <h1>
    Hello, World!
  </h1>
  <span>
    0
     - 
    0e+0
     -
     
    0.00000000000000000
  </span>
</div>
`;
